src/index.tsx

```diff
@@ -24,7 +24,10 @@ import AppProviders from './context';
 
 initGoogleTagManager();
 initLogRocket();
+
+// Setup immer
 enableMapSet();
+setAutoFreeze(false);
```

Comment:
This allows AJV to more easily manipulate the data being passed to it. Related to this change but one I have been eyeing for a while so made it here.